src/backend/common/helpers/event_short_name_helper.py

```diff
@@ -72,8 +101,7 @@ def get_short_name(cls, name_str: str, district_code: Optional[str] = None) -> s
                 result = match.group(1).strip()
             else:
                 result = short.strip()
-            if result.startswith("FIRST"):
-                result = result[5:]
-            return result.strip()
+            result = re.sub("FIRST( in)?", "", result, flags=re.IGNORECASE)
+            return " ".join(result.split())
```

Comment:
I’m going to double check this and make sure we’re not going to accidentally remove “first” if it’s part of a word.

Reply:
Updated 👍